internal/iptables/iptables.go

```diff
@@ -697,11 +697,11 @@ func getIPTablesRestoreWaitFlag(version *utilversion.Version, exec utilexec.Inte
 	// --version, assume it also supports --wait
 	vstring, err := getIPTablesRestoreVersionString(exec, protocol)
 	if err != nil || vstring == "" {
-		klog.V(3).InfoS("Couldn't get iptables-restore version; assuming it doesn't support --wait")
+		logrus.Debug("Couldn't get iptables-restore version; assuming it doesn't support --wait")
 		return nil
 	}
 	if _, err := utilversion.ParseGeneric(vstring); err != nil {
-		klog.V(3).InfoS("Couldn't parse iptables-restore version; assuming it doesn't support --wait")
+		logrus.Debug("Couldn't parse iptables-restore version; assuming it doesn't support --wait")
 		return nil
 	}
 	return []string{WaitString}
```

Comment:
Here also, the log level would probably be better served as a warning to the user.